python/pyspark/sql/connect/functions.py

```diff
@@ -16,14 +16,247 @@
 #
 from pyspark.sql.connect.column import Column, LiteralExpression, ColumnReference
 
-from typing import Any
+from typing import Any, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pyspark.sql.connect._typing import ColumnOrName
 
 # TODO(SPARK-40538) Add support for the missing PySpark functions.
 
 
+def _to_col(col: "ColumnOrName") -> Column:
+    return col if isinstance(col, Column) else column(col)
+
+
 def col(x: str) -> Column:
     return Column(ColumnReference(x))
 
 
+column = col
```

Comment:
why???

Reply:
`column` itself is a public function.

https://github.com/apache/spark/blob/master/python/pyspark/sql/functions.py#L208